packages/clay-date-picker/src/InputDate.tsx

```diff
@@ -55,7 +58,9 @@ const ClayDatePickerInputDate: React.FunctionComponent<IProps> = ({
 		<>
 			<input name={inputName} type="hidden" value={memoizedValue} />
 			<ClayInput
+				{...otherProps}
 				aria-label={ariaLabel}
+				disabled={disabled}
```

Comment:
Since we are already using `{...otherProps}`, we don't need to explicitly set `disabled` or use it off of the props anymore

Reply:
We can also probably remove other explicit props like `placeholder` and `onChange`. Unless they are used elsewhere.

Reply:
Oh yeah, I'll clean it up.